src/System.IO.Pipes/src/System/IO/Pipes/PipeAsyncResult.cs

```diff
@@ -59,6 +53,11 @@ public bool CompletedSynchronously
             get { return _completedSynchronously; }
         }
 
+        internal ManualResetEventSlim WaitHandle
```

Comment:
Calling this WaitHandle is confusing to me, as a ManualResetEventSlim isn't actually a WaitHandle.